Resources/views/CRUD/list_orm_many_to_one.html.twig

```diff
@@ -14,7 +14,7 @@ file that was distributed with this source code.
 {% block field %}
     {% if value %}
         {% set route_name = field_description.options.route.name %}
-        {% if field_description.hasAssociationAdmin and field_description.associationadmin.hasRoute(route_name) and field_description.associationadmin.hasAccess(route_name, value) %}
+        {% if not field_description.options.identifier|default(false) and field_description.hasAssociationAdmin and field_description.associationadmin.hasRoute(route_name) and field_description.associationadmin.hasAccess(route_name, value) %}
```

Comment:
Please linebreak this, it still wouldn't fit if the GH editor was twice as large.